data/wifi.html

```diff
@@ -1,7 +1,14 @@
 <form method="post" action="/wifi">
-        <p>WiFi Network <b>to connect to</b><br><input placeholder="Type or Select your WiFi Network" value="%SSID%"
-                        name="s"></p>
-        <p><label><b>WiFi Password</b></label><br><input type="password" placeholder="Enter your WiFi Password"
-                        value="%PASS%" name="p"></p>
-        <button>Save</button>
+  <fieldset>
+    <legend>WiFi Network Configuration:</legend>
+    <p><label>Network Name (SSID)</label><br>
```

Comment:
can you lease make the description a little more verbose, accentuating the fact that this isn't the network that Owie itself advertises but is a network it has to connect to?

Maybe add a description that it's unlikely anyone will have to use this functionality, it's mostly useful for developers.